src/components/MenuButton/MenuButton.css

```diff
@@ -0,0 +1,58 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+
+.y-menu-button {
+  height: 20px;
+  width: 20px;
+  border: 0;
+  padding: 0;
+}
+
+.y-menu-button > .ms-Button-flexContainer > .y-icon {
+  top: 0;
+}
+
+/* Overall top/bottom menu padding */
+.y-menu-button--callout .ms-ContextualMenu-list {
+  padding: .4rem 0;
+}
+
+/* Element styling overrides */
+.y-menu-button--callout .ms-ContextualMenu-link,
+.y-menu-button--callout .ms-ContextualMenu-header {
+  padding: 0;
+}
+
+.y-menu-button--item {
+  padding: .4rem 1.2rem .8rem
+}
+.y-menu-button--item-header {
+  padding: .6rem 1.2rem 1rem
+}
+
+.y-menu-button--callout .ms-ContextualMenu-item:hover .ms-ContextualMenu-link,
+.y-menu-button--callout .ms-ContextualMenu-item:active .ms-ContextualMenu-link,
+.y-menu-button--callout .ms-ContextualMenu-item:focus .ms-ContextualMenu-link {
+  background-color: $dropdownOptionBackgroundColor__focused;
+}
+
+/* Headers and Dividers */
+.y-menu-button--callout .ms-ContextualMenu-header {
+  background-color: initial;
+}
+
+.y-menu-button--callout .ms-ContextualMenu-divider {
+  background-color: $borderColor__secondary;
+  margin-bottom: -1px;
+}
+
+
+.y-menu-button--callout a:hover {
```

Comment:
we should replace these `a` tag selectors with the classname of the `a` and `button` tags. also want to make sure the bgcolors are correctly the same on both tags

Reply:
These a tags have the same class as all the items, .ms-ContextualMenu-link. I replaced the a selector with that class and I think it should be okay.

Reply:
bgcolors look the same for link and non link